app/src/ui/branches/pull-request-badge.tsx

```diff
@@ -16,22 +16,22 @@ export class PullRequestBadge extends React.Component<
   {}
 > {
   public render() {
+    const status = this.props.status
+
+    if (!status || status.totalCount === 0) {
+      return null
+    }
+
     return (
       <div className="pr-badge">
         <span className="number">#{this.props.number}</span>
 
-        {this.renderPullRequestStatus()}
+        {this.renderPullRequestStatus(status)}
       </div>
     )
   }
 
-  private renderPullRequestStatus() {
-    const status = this.props.status
-
-    if (!status || status.totalCount === 0) {
-      return null
-    }
-
-    return <CIStatus status={status} />
-  }
+  private renderPullRequestStatus = (status: PullRequestStatus) => (
```

Comment:
Seems a little silly to have this function when it doesn't do anything 😄

Reply:
But those are the easiest to test! 😏